subsys/net/l2/openthread/openthread.c

```diff
@@ -343,6 +343,13 @@ int openthread_start(struct openthread_context *ot_context)
 		otLinkSetPollPeriod(ot_context->instance, OT_POLL_PERIOD);
 	}
 
+	NET_INFO("OpenThread version: %s", otGetVersionString());
+
+	if (!IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR_RCP)) {
```

Comment:
Is the condition correct here? I'd rather expect something like 
```
if (IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR)) {
	NET_DBG("OpenThread co-processor.");
	goto exit;
}
```
Let's move this at the beginning of the function (before `if (IS_ENABLED(CONFIG_OPENTHREAD_MTD_SED)) {`)

Reply:
Done.